src/chapters/14-provider-components.md

```diff
@@ -48,7 +48,7 @@ Awesome, one step done. Now, this input looks great, but it doesn't actually *do
 
 In order to make our search box actually work, we are going to need to retain and store the text that the user types in when they use the search box. This text is the search query, and it is a piece of *[state][TODO: link to state]* that is going to change whenever the user types something into the search box.
 
-But, where are we going to put this newly-introduced piece of state? In order to wire up the search box, we need a place to store the search query. At the moment, our search box lives on the `index.hbs` route template, which doesn't have a good place to store this search query state. Darn, this would be so much easier to do if we had a component, because we could just store the state directly on the component!
+But where are we going to put this newly-introduced piece of state? In order to wire up the search box, we need a place to store the search query. At the moment, our search box lives on the `index.hbs` route template, which doesn't have a good place to store this search query state. Darn, this would be so much easier to do if we had a component, because we could just store the state directly on the component!
 
 Wait...why don't we just refactor the search box into a component? Once we do that, this will all be a bit easier&mdash;hooray!
```

Comment:
Something something spaces around emdash (I think we settled on spaces around it)

Reply:
I ended up [changing it](https://github.com/ember-learn/super-rentals-tutorial/pull/75/commits/d6fec45a20c626f347a2d880f8cb87ced7689a24) to have no spaces because it's actually...grammatically incorrect to do that. I kept spaces because it was easier to read while we were writing the prose...but now I changed it 😬